client/src/graphics/draw.rs

```diff
@@ -289,7 +289,7 @@ impl Draw {
         let draw_started = Instant::now();
         let view = sim.as_ref().map_or_else(Position::origin, |sim| sim.view());
         let projection = frustum.projection(1.0e-4);
-        let view_projection = projection.matrix() * math::mtranspose(&view.local);
+        let view_projection = projection.matrix() * <math::MIsometry<_> as Into<na::Matrix4<_>>>::into(view.local.mtranspose());
```

Comment:
This should be `na::Matrix4::<_>::from(view.local.mtranspose())`, and similarly in the other cases below. Using UFCS to invoke `Into::into` is usually far more verbose than necessary.